tests/main/interfaces-timeserver-control/task.yaml

```diff
@@ -17,6 +16,13 @@ prepare: |
     # Install a snap declaring a plug on timeserver-control
     install_local test-snapd-timedate-control-consumer
 
+    if [[ "$SPREAD_SYSTEM" == ubuntu-19.10-* ]]; then
+        # ensure chrony is not installed as it interferes with
+        # the systemd-timesyncd.service
+        apt remove -y chrony
```

Comment:
Slightly uneasy about removing the package, perhaps we should install it in restore?

Reply:
Experimenting a bit with automatically restoring packages in #7414